[Samsung Galaxy] fix naming for W20/21 5G

adriens opened the pull request:
# :grey_question: About

Rename [`Samsung Galaxy W21 5G`](https://www.kimovil.com/en/where-to-buy-samsung-galaxy-w21-5g) to `Galaxy W21 5G`

![image](https://user-images.githubusercontent.com/5235127/214170322-ce49bd7c-76e4-4044-accf-5bac6508cf10.png)

# :tickets: Related stuff

- https://github.com/endoflife-date/endoflife.date/pull/2353
- https://github.com/endoflife-date/endoflife.date/pull/2336
- https://github.com/endoflife-date/endoflife.date/pull/2156

adriens committed:
fix(Samsung Galaxy) : Samsung Galaxy W21 5G

adriens committed:
Galaxy W20 5G

adriens reviewed (commented):
see [Samsung Galaxy W20 5G ](https://www.kimovil.com/en/where-to-buy-samsung-galaxy-w20-5g) : 

![image](https://user-images.githubusercontent.com/5235127/214171145-622ab4ce-d91b-4565-bd2d-d5921b02de9f.png)

usta commented on products/samsungmobile.md:
```diff
@@ -416,7 +416,7 @@ releases:
     releaseDate: 2019-12-18
 
--   releaseCycle: "W20 5G"
+-   releaseCycle: Galaxy W20 5G"
```
Please add missing `"`

usta reviewed (changes requested):
Please add missin double quote `"` for Galaxy W20 5G releaseCycle

adriens committed:
Typo

cf https://github.com/endoflife-date/endoflife.date/pull/2357#pullrequestreview-1266785953

adriens committed:
"Samsung W21 5G" -> "W21 5G"

cf https://github.com/endoflife-date/endoflife.date/pull/2357#issuecomment-1407168639
